caffe2/sgd/adadelta_op.h

```diff
@@ -0,0 +1,179 @@
+#pragma once
```

Comment:
Please use #ifndef instead of #pragma once to guard the h file.
https://google.github.io/styleguide/cppguide.html#The__define_Guard